rust/template/differential_datalog/src/ddlog.rs

```diff
@@ -23,6 +23,10 @@ pub trait DDlogInventory {
     /// Convert a `RelId` into its symbolic name.
     fn get_table_name(&self, tid: RelId) -> Result<&'static str, String>;
 
+    /// Given a table name, the the original name (from the 'original' DDlog
```

Comment:
As you commented below, this always returns a value, so I will keep it Result. The String is there because all other functions in this group have the same signature,. BTW: we may probably want to add this to the C API too.

Reply:
> As you commented below, this always returns a value, so I will keep it Result

In that case, documentation should be more clear about this.

> BTW: we may probably want to add this to the C API too.

As well as Java and Go :-/